WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorView.swift

```diff
@@ -53,54 +32,64 @@ struct CustomFieldEditorView: View {
             VStack(alignment: .leading, spacing: Layout.sectionSpacing) {
                 // Key Input
                 VStack(alignment: .leading, spacing: Layout.subSectionsSpacing) {
-                    Text("Key") // todo-13493: set String to be translatable
+                    Text(Localization.keyLabel)
                         .foregroundColor(Color(.text))
                         .subheadlineStyle()
 
-                    TextField("Enter key", text: $key) // todo-13493: set String to be translatable
-                        .bodyStyle()
+                    TextField(Localization.keyPlaceholder, text: $key)
+                        .foregroundColor(Color(.text))
+                        .subheadlineStyle()
                         .padding(insets: Layout.inputInsets)
                         .background(Color(.listForeground(modal: false)))
                         .overlay(
                             RoundedRectangle(cornerRadius: Layout.cornerRadius).stroke(Color(.separator))
                         )
-                        .onChange(of: key) { _ in
-                            checkForModifications()
-                        }
                 }
 
                 // Value Input
                 VStack(alignment: .leading, spacing: Layout.subSectionsSpacing) {
                     HStack {
-                        Text("Value") // todo-13493: set String to be translatable
+                        Text(Localization.valueLabel)
                             .foregroundColor(Color(.text))
                             .subheadlineStyle()
 
                         Spacer()
 
                         if !isReadOnlyValue {
-                            Button(action: {
-                                showRichTextEditor = true
-                            }) {
-                                Text("Edit in Rich Text Editor") // todo-13493: set String to be translatable
-                                    .font(.footnote)
+                            Picker(Localization.editorPickerLabel, selection: $showRichTextEditor) {
+                                Text(Localization.editorPickerText).tag(false)
+                                Text(Localization.editorPickerHTML).tag(true)
                             }
-                            .buttonStyle(.plain)
-                            .foregroundColor(Color(uiColor: .accent))
+                            .pickerStyle(.segmented)
+                            .fixedSize()
```

Comment:
❓ Any reason to use `fixedSize()` here? From my experience, this modifier brings more issues than fixes - it can cause unexpected layout behaviors like stretched views.

Reply:
without `.fixedSize()`, `.segmented` picker style by default stretches itself to all available width:
<img src="https://github.com/user-attachments/assets/c926406f-b11f-465f-8ec1-de12de1c357f" width="42%" />

This looks especially bad on wider screen:

<img src="https://github.com/user-attachments/assets/239e193a-4498-464e-b488-9d64876ffe40" width="42%" />

Reply:
You can make the Picker smaller by letting the "Value" text have the maximum width possible with `.frame(maxWidth: .infinity, alignment: .leading)`.

Reply:
Thanks, that's a good solution. https://github.com/woocommerce/woocommerce-ios/pull/13925/commits/fca268c1a8d5570dfdb2c98858853897166bde67

I wasn't aware of the issue with `fixedSize()`, thanks for mentioning that.